crypto/client_test.go

```diff
@@ -0,0 +1,63 @@
+package crypto
+
+import (
+	"testing"
+)
+
+func TestNew(t *testing.T) {
+	fixtures := []struct {
+		t  string
+		fn func(interface{}) bool
+	}{
+		{
+			t: "",
+			fn: func(c interface{}) bool {
+				_, ok := c.(*ChaCha20Poly1305Client)
+				return ok
+			},
+		},
+		{
+			t: "asdf",
+			fn: func(c interface{}) bool {
+				_, ok := c.(*ChaCha20Poly1305Client)
+				return ok
+			},
+		},
+		{
+			t: "chachapoly",
```

Comment:
Why not use `ConfigTypeChaChaPoly` (applies to all `t`s below as well)?

Reply:
Yeah, using the consts would make a lot of sense 😆